docs/contributing/style-guide.md

```diff
@@ -7,34 +7,13 @@ tags: [style-guide,contributing]
 
 # Rust Style Guide
 
-## Motivation - why not rely on a formatting tool for code style?
-
-The code style is way more than just formatting. In many cases formatting can be
-automated. According to rustfmt docs: "formatting code is a mostly mechanical
-task which takes both time and mental effort. By using an automatic formatting
-tool, a programmer is relieved of this task and can concentrate on more
-important things.". While in many cases it is true, if the code author does not
-take extra effort to make his code pretty by refactoring long lines to
-variables, moving code to specific modules, or sections, the formatting tool
-will result in code that is hard to read and hard to write. Thus, it is
-important to take time to write code in such way that we can be proud of its
-quality.
-
-Because `rustfmt` does not support many of our requirements, we have created a
-guide to describe how to format Rust code in this project. Please read it
-carefully. We hope that in the future, some of the things described below will
-be possible while using `rustfmt` (and we encourage you to contribute there!),
-however, even if it happens, many parts of this guide will still be valid and
-will need to be handled manually.
-
+We are using `rustfmt` for the basic formatting of our rust code, which is also
+checked on CI. We have some additional guidelines for imports, naming, sections
+within files and naming which are documented here.
```

Comment:
I see later in this file, that the examples code are not formatted with rustfmt.